app/models/presenters/stock_plate_presenter_with_info.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+#
+module Presenters
+  # A stock plate presenter with additional informational messages.
+  # This subclass of StockPlatePresenter adds specific informational messages
+  # to provide additional context information to the user.
+  class StockPlatePresenterWithInfo < StockPlatePresenter
+    def initialize(*args)
+      super
+      messages = purpose_config.dig(:presenter_class, :args, :messages)
+      messages&.each { |message| add_info_message(message) }
```

Comment:
Thanks, I think this is nice, to pull it from the config. To take it one step further - is there actually any need for a new presenter? Would be cool to have this functionality on all presenters?

Reply:
I thought about it and I was not sure. I will update it in base class. 👍

Reply:
@KatyTaylor updated 👍